### DISCUSSION TITLE
Does GPS Time Sync Utilize PPS from Onboard Receiver?

### DISCUSSION THREAD

**Discussion by AlexLandherr** [gopro/labs#563] [General]
I have a somewhat technical question about GPS Time Sync on the HERO11 Black.

When the camera takes timing info from its GPS receiver does it or can it utilize PPS (Pulse Per Second) for extra accuracy?

Some low-accuracy systems just use serial data output by e.g. USB GPS receivers while some used for Raspberry Pi:s use PPS to get the start of a second and then use serial data to determine which second.

**Comment by dnewman-gpsw** [comment]
In the Labs GPS Time Sync, it does attempt to enable PPS (timepulse internally), using an interrupt for when the time is ready.  Precision is improved, but I think it could be even better with a deeper hack.

**Reply by AlexLandherr** [reply]
If any of the developers/contributors could look into that I think it would benefit those needing very good synchronization of multiple cameras or just want really accurate time stamping of their footage and images.

My experience is not with embedded development, but in backend development for servers using Java.

**Reply by dnewman-gpsw** [reply]
I'm one of those developers.  GPS Sync is at best +/-50ms accurate on HERO11, good enough for most uses. Any more precision was going to be costly in engineering time.

**Reply by AlexLandherr** [reply]
Ok, thanks for letting me know.
